deepchem/models/wandblogger.py

```diff
@@ -0,0 +1,121 @@
+import copy
+import logging
+import importlib.util
+from typing import Optional, Union
+
+logger = logging.getLogger(__name__)
+
+
+def is_wandb_available():
+  return importlib.util.find_spec("wandb") is not None
+
+
+class WandbLogger(object):
+  """Weights & Biases Logger for KerasModel.
```

Comment:
It should work with all models, not just KerasModel.  We also have a TorchModel class, and we're in the process of creating a JAXModel class.

Reply:
Will do! We'll also add the integration to TorchModel shortly.